client/lib/feed-stream-store/index.js

```diff
@@ -19,22 +20,25 @@ const wpcomUndoc = wpcom.undocumented();
 function feedKeyMaker( post ) {
 	return {
 		feedId: post.feed_ID,
-		postId: post.ID
+		postId: post.ID,
+		localMoment: moment( post.date ).local(),
```

Comment:
looks like you don't really need the `.local()` calls https://momentjs.com/docs/#/parsing/utc/

We'd have to use `.utc()` explicitly to get into utc mode.

<img width="351" alt="developer_tools_-_https___momentjs_com_docs_" src="https://cloud.githubusercontent.com/assets/14350/23968335/b2369510-0998-11e7-8159-043f6244ea0c.png">